[DOCS-1476] Document how to create a project (#1309)
wandb/docs: content/guides/models/track/project-page.md

The versions tab shows all versions of the artifact as well as columns for each of the numeric values of the Run History at the time of logging the version. This allows you to compare performance and quickly identify versions of interest.

## Create a project
Create a project from the **Projects** page.

> 1. In the W&B App, click the icon in the upper left. In the sidebar's **Projects** section, click **View all**. 

Reviewer: Which icon?

Reviewer: You can also create a project on a team's landing page and you can programmatically create a project. For the latter, W&B creates a project for you if you specify a name in wandb.init(project="") that doesn't exist.

Author: PTAL, I expanded these details and tried to be more clear about "which icon". It's the global navigation icon.

1. Click **Create new project**.
1. Set **Team** to the name of the team that will own the project.

> 1. Set **Name** to a name for the project.

Reviewer: This sounded awkward to me. Maybe rephrase to:

```suggestion
1. Specify a name for your project using the **Name** field. 
```

From the **Projects** page:
1. Click the global navigation icon in the upper left. The navigation sidebar opens.
1. In the **Projects** section of the navigation, click **View all** to open the project overview page.
1. Click **Create new project**.
1. Set **Team** to the name of the team that will own the project.
1. Specify a name for your project using the **Name** field. 
1. Set **Project visibility**, which defaults to **Team**.
1. Optionally, provide a **Description**.
1. Click **Create project**.

From a team's landing page:
1. Click the global navigation icon in the upper left. The navigation sidebar opens.
1. In the **Teams** section of the navigation, click the name of a team to open its landing page.
1. In the landing page, click **Create new project**.
1. **Team** is automatically set to the team that owns the landing page you were viewing. If necessary, change the team.
1. Specify a name for your project using the **Name** field. 
1. Set **Project visibility**, which defaults to **Team**.
1. Optionally, provide a **Description**.
1. Click **Create project**.

   {{% /tab %}}

>    {{% tab header="wandb.init()" %}}

Reviewer: Suggestion so it's consistent with the rest of the docs.

```suggestion
   {{% tab header="Python SDK" %}}
```

> To create a project programmatically, specify a `project` when calling `wandb.init()`. If the project does not yet exist, it is created automatically. Refer to the [`wandb.init()` API reference]({{< relref "/ref/python/init/#examples" >}}).

Reviewer: Let's add a simple code snippet to this tab and link to the API Reference docs for more info on additional wandb.init() parameters. Something like:

```python
import wandb

with wandb.init(entity="<entity>", project="<project_name>") as run:
   run.log({"accuracy": .95})
```

Author: Done!
